astropy/wcs/src/pyutil.c

```diff
@@ -340,11 +340,21 @@ wcserr_fix_to_python_exc(const struct wcserr *err) {
 }
 
 void
-wcshdr_err_to_python_exc(int status) {
+wcshdr_err_to_python_exc(int status, const struct wcsprm *wcs) {
+  /* Add error to wcslib error buffer */
+  wcsperr(wcs, "");
```

Comment:
```suggestion
  wcsperr(wcs, NULL);
```
From the [docs](https://www.atnf.csiro.au/people/mcalabre/WCS/wcslib/wcs_8h.html#a8fe5dcd9927240dc0348b850ee662367):
> prefix | If non-NULL, each output line will be prefixed with this string.